session.go

```diff
@@ -1248,6 +1260,14 @@ func (q *Query) reset() {
 	*q = Query{}
 }
 
+var (
+	lwtRegexp = regexp.MustCompile("(.*IF( *)(NOT)?( *)EXISTS$)|((.*)IF( *)(.*)=(.*))")
```

Comment:
This expression does not work in many ways
* It gets triggered in `CREATE` statements that have `IF NOT EXIST` 
* It is case sensitive
* We should use \s for spaces, not sure how C* reacts to tabs though
* `*` matches zero occurrences it should be sth like \s+IF\s+(NOT\s+)? EXISTS\s+$
* Matching [a "condition"](http://cassandra.apache.org/doc/latest/cql/dml.html#update) is much more difficult

I think that maybe it would be easier to find an unquoted `<space>IF<space>`?

@kostja what do you think?

Reply:
Ill expand the regex and some test cases to match.

Reply:
This seems to work better: `(INSERT|UPDATE).*\s+IF\s+(NOT)?\s*EXISTS$|\s+IF\s+.*=.*`

Reply:
A little safer perhaps: `(INSERT|UPDATE).*\s+(IF\s+(NOT)?\s*EXISTS$|IF\s+.*=.*)`

Reply:
Still `INSERT INTO X (A, B) VALUES (" IF ", 42)` WHERE A = 10` triggers it...

Reply:
Not relevant after using the metadata bit.